packages/algob/src/internal/core/errors.ts

```diff
@@ -190,16 +190,16 @@ function _applyErrorMessageTemplate (
 export function parseAlgorandError (e: RequestError, ctx: AnyMap): Error {
   if (e === undefined) { return new BuilderError(ERRORS.NETWORK.NODE_IS_NOT_RUNNING); }
 
-  if (e?.statusCode !== undefined) {
-    if (e.statusCode >= 400 && e.statusCode < 500) {
+  if (e.response?.statusCode !== undefined) {
+    if (e.response?.statusCode >= 400 && e.response?.statusCode < 500) {
       return new BuilderError(ERRORS.ALGORAND.BAD_REQUEST, {
-        status: e.statusCode,
-        message: e.body?.message ?? e.text,
+        status: e.response?.statusCode,
+        message: e.response?.body?.message ?? e.response?.text,
```

Comment:
please add a check for `e.error.text` (or `e.response.error.text` - not sure which one is correct) as a third option if the 2 previous are not defined.

Reply:
Sorry, i don't follow. I think the `response` part was added after the latest sdk update. Line 197 is checking both possibilites (`body.message` and `text`). 
BDW, this block will execute only if the error is `Bad Request`.

Reply:
I was thinking about:
```
e.response?.body?.message ?? e.response?.text ?? e.response?error
```
Although I don't remember if the last part is `e.response?error` or `e.error`

Reply:
ok. It's `e.response?error`